controls: refine `loop_anchor` description, clarify range

ronso0 opened the pull request:
`loop_anchor`
`beatloop_rN_toggle`
`beatloop_rN_activate`
`beatlooproll_rN_activate`

added in https://github.com/mixxxdj/mixxx/pull/12745

Closes #715 

Preview: [here](https://deploy-preview-719--mixxx-manual.netlify.app/chapters/appendix/mixxx_controls#control-[ChannelN]-beatloop_rX_activate) and below

Holzhaus commented:
Hi, I'm a bit confused because I don't see `loop_anchor` in the diff. Instead, I see `beetloop_size` which is not mentioned in the PR description.

ronso0 commented:
:thinking: I remember I typed that but it got lost somehow. Fixed.

> Instead, I see beetloop_size which is not mentioned in the PR description.

I moved `beatloop_size` up to consolidate the `_activate`/`_toggle` controls.

acolombier reviewed (approved):
LGTM! Thanks for adding this missing CO

acolombier commented:
Letting @Holzhaus a chance to check this PR again before pressing merge...

ronso0 commented:
Ready, steady, go?

Swiftb0y reviewed (commented):
This is not fully self-explanatory for someone that doesn't know this feature at all.

Swiftb0y reviewed (approved):
Thank you. Sorry for the unnecessary comments on the other sections.
